cmd/manager/main.go

```diff
@@ -37,12 +37,20 @@ import (
 func main() {
 	klog.InitFlags(nil)
 	flag.Set("logtostderr", "true")
+	watchNamespace := flag.String("namespace", "",
+		"Namespace that the controller watches to reconcile cluster-api objects. If unspecified, the controller watches for cluster-api objects across all namespaces.")
 	flag.Parse()
 
 	cfg := config.GetConfigOrDie()
 
 	// Setup a Manager
-	mgr, err := manager.New(cfg, manager.Options{})
+	var opts manager.Options
+	if *watchNamespace != "" {
+		opts.Namespace = *watchNamespace
+		klog.Infof("Watching cluster-api objects in only in namespace %q for reconciliation.", opts.Namespace)
```

Comment:
```suggestion
		klog.Infof("Watching cluster-api objects only in namespace %q for reconciliation.", opts.Namespace)
```

Reply:
derp.. 😝 fixed it. sorry for not catching that myself